### ISSUE TITLE
scaffold 

### ISSUE THREAD

**Issue by ataniii** [lambda-client/lambda-legacy#165]
can you redo the scaffold it is super buggy

**Comment by master7720** [comment]
facts

**Comment by Doogie13** [comment]
[My branch](https://github.com/Doogie13/lambda/actions/runs/1334096773) has a rewritten scaffold but is pending testing. could you try this and tell me if it works?

**Comment by czho** [comment]
Looks good, does it have rotations?

**Comment by Doogie13** [comment]
> Looks good, does it have rotations?

yes, but they are currently kinda trash (doesn't work at all on strict 2bpvp) so ima use the built-in ones rather than just sending another cpacketplayer.rotation on top of existing ones

**Comment by ToxicAven** [comment]
@Doogie13 is this scaffold rewrite still in development?

**Comment by Doogie13** [comment]
> @Doogie13 is this scaffold rewrite still in development?

No. It's terrible and needs to be completely rewritten if I am to push it

**Comment by ToxicAven** [comment]
Alrighty. No problem!

**Comment by mmvanheusden** [comment]
Good news: a new scaffold is currently under development!
